ironfish-cli/src/commands/status.ts

```diff
@@ -40,14 +40,20 @@ export default class Status extends IronfishCommand {
     const screen = blessed.screen({ smartCSR: true, fullUnicode: true })
     const statusText = blessed.text()
     screen.append(statusText)
+    let previousResponse: GetNodeStatusResponse | null = null
 
     // eslint-disable-next-line no-constant-condition
     while (true) {
       const connected = await this.sdk.client.tryConnect()
 
       if (!connected) {
         statusText.clearBaseLine(0)
-        statusText.setContent('Node: STOPPED')
+        if (previousResponse != null) {
```

Comment:
You can probably shorten this:

```suggestion
        if (previousResponse) {
```